### PR TITLE
Add option to set booleans right here with selinux

### PR DESCRIPTION
**Pull request by jcpunk**
It makes more sense to me to set some selinux booleans here.  Not all of them, but a few look right to me right here.

### CODE REVIEW

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -142,4 +142,17 @@
...
+  describe 'when setting booleans' do
+    let(:params) do
+      { :mode      => 'enforcing',
+        :type      => 'strict'
+        :setsebool => {'nfs_export_all_ro' => { 'value' => 'on' } }
+      }
+    end
+
+    it { should contain_class('selinux') }
+    it { should contain_class('selboolean') }
```
Comment (ghoneycutt) [inline spec/classes/init_spec.rb]: setboolean is not a class it is a resource type

**Comment by ghoneycutt** [comment]
Thanks, I like this!

What you are doing is done in my types module. This should use hiera_hash() by default like packages does in types. You can also see how to do the spec tests from there.

https://github.com/ghoneycutt/puppet-module-types

**Comment by ghoneycutt** [comment]
I would keep it with this module if we had a defined type called selinux::bool or whatever, but since it is a native type it should be part of my types module. Could you submit a PR there?

**Comment by jcpunk** [comment]
PR submitted over in the types module.  Closing this one.

**Comment by ghoneycutt** [comment]
Thank you!

**Pull request closed without merging**